info.yml

```diff
@@ -10,6 +10,11 @@
 
 configuration:
 
+    display_name:
+        type: str
+        default_value: Scene Breakdown2
```

Comment:
I would like this default value to be `Scene Breakdown` without the 2 as this is how it will be included by default. The precedent exists with both `workfiles` and `loader` as both went to version 2 of the actual TK apps but still appear in the menus as Open / Save / Load. Thanks for re-opening the PR!

Reply:
Allright, I'll get that done!

Reply:
Thank you!